app.py

```diff
@@ -267,6 +267,8 @@ def _push_image(
 
     image_name = image.rsplit("/", maxsplit=1)[1]
     if "quay.io" in push_registry:
+        if "sha256" in image_name:
```

Comment:
I think the `if` statement is redundant, the replace will have no effect if '@sha256' (sha256 is a substring) would be found.

Reply:
Thanks for pointing it out, shouldn't have added the redundant if there.